OpenRA.Game/Orders/UnitOrderGenerator.cs

```diff
@@ -57,9 +58,10 @@ public IEnumerable<Order> Order(World world, CPos xy, MouseInput mi)
 		public string GetCursor(World world, CPos xy, MouseInput mi)
 		{
 			var useSelect = false;
-			var underCursor = world.ScreenMap.ActorsAt(mi)
+			var worldPixel = world.ScreenMap.Viewport.ViewToWorldPx(mi.Location);
```

Comment:
It would be more appropriate to change the `GetCursor` signature to pass in the viewport or the worldrenderer.